next/test/validation/boolean_schema.test.ts

```diff
@@ -0,0 +1,18 @@
+import { describe, expect, it } from '@jest/globals'
+import { createHeadlessForm } from '../../src'
+
+describe('boolean schema validation', () => {
+  it('returns an error if the value is false', () => {
+    const form = createHeadlessForm({ properties: { name: false } })
+
+    expect(form.handleValidation({ name: 'anything' })).toMatchObject({ formErrors: { '.name': 'always fails' } })
```

Comment:
`formErrors: { '.name': 'always fails' }` does not match what v0 returns. I'll fix that in another commit.